frontend/src/container/ListOfDashboard/DashboardsList.tsx

```diff
@@ -311,12 +301,15 @@ function DashboardsList(): JSX.Element {
 
 	const handleSearch = (event: ChangeEvent<HTMLInputElement>): void => {
 		setIsFilteringDashboards(true);
-		setSearchValue(event.target.value);
 		const searchText = (event as React.BaseSyntheticEvent)?.target?.value || '';
-		const filteredDashboards = filterDashboard(searchText, dashboardListResponse);
+		const filteredDashboards = filterDashboard(
+			searchText,
+			dashboardListResponse || [],
```

Comment:
You removed this `data: dashboardListResponse = [],` on top and using `[]` for occurences, any specific reason?

Reply:
everytime the component re-renders there is always a new reference for the `dashboardListResponse`. hence causing all the useEffects to trigger again. We do not want that hence never initialise the default values here!